types/wordpress__components/card/index.d.ts

```diff
@@ -0,0 +1,32 @@
+import { StyledComponent } from '@emotion/styled';
+
+declare namespace Card {
+    type CardSize = 'large' | 'medium' | 'small' | 'extraSmall';
+    interface Props {
+        /**
+         * `className` of the container.
+         */
+        className?: string;
+
+        /**
+         * Renders a borderless card.
+         */
+        isBorderless?: boolean;
+
+        /**
+         * Renders an elevated card.
+         */
+        isElevated?: boolean;
+
+        /**
+         * Determines the amount of padding within the card.
+         *
+         * @default "medium"
```

Comment:
This tag should be [`@defaultValue`](https://github.com/microsoft/tsdoc/blob/59860dd8f3ebd2a0e4441c392482472937cf356b/tsdoc/src/details/StandardTags.ts#L60-L73)

Reply:
1cd2016de9